demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -9,10 +9,20 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
     toolbarActionProp: true,
     rowActionProp: true,
     columns: [
-        { type: "boolean", name: "inStock", headerName: "In stock", width: 90 },
-        { type: "text", name: "title", headerName: "Titel", minWidth: 200, maxWidth: 250 },
-        { type: "text", name: "description", headerName: "Description" },
+        {
+            type: "combination",
+            name: "overview",
+            headerName: "Overview",
+            minWidth: 200,
+            maxWidth: 250,
+            primaryText: "title",
+            secondaryText: "description",
+            visible: (breakpoints) => breakpoints.down("md"),
```

Comment:
Why is this a function? For type safety?

Reply:
For type safety and for easy access to the themes breakpoints. 
This also allows setting the breakpoint in the same way you'd do in other places with MUI, e.g., the handmade grid:

https://github.com/vivid-planet/comet/blob/02a4da594177936cc243daea72a6df8cfa77df48/demo/admin/src/products/ProductsGrid.tsx#L80

An alternative would be to set `visible` to an object like this:

```ts
visible: {
    direction: "down",
    breakpoint: "md",
},
```
```ts
visible: {
    direction: "up",
    breakpoint: "md",
},
```
```ts
visible: {
    direction: "between",
    start: "md",
    end: "lg",
},
```

Reply:
We could also achieve type safety with advanced types: https://www.typescriptlang.org/play/?#code/C4TwDgpgBAIglgJwgY2HA9gOygXigIgFcx8oAfAgE3QHdNSL8AjCYGiCegbgFgAoUJCgBpCCFxQA5PgC2lfJPJT8AGwDmCpZkIyWCXn36DoANTgBnOExXQ8AA2AALCDIgA6JkgCGAazDo4TGBzNwASAG94JFQMTABfAAoI0RA4gEo7A35kLHNgKC8ALigzS2tbKScXd08IX39A4I9Wdk4E1Q00yWzc-KZi0qsbCUkq1w9vPwCgkJY2Dkx25xUVdHwuoA

@nsams what do you think? Is a function in the config okay?

Reply:
> maybe like you advanced type but a bit [shorter](https://www.typescriptlang.org/play/?#code/C4TwDgpgBAIglgJwgY2HA9gOygXigIgFcx8oAfAgE3QHdNSL8AjCYGiCegbgFgAoUJCgBpCCFxQA5PgC2lfJPJT8AGwDmCpZkIyWCXn36DoANTgBnOExXQ8AAwAkAb3hJUGTAF8AFM9EhPAEo7A35kLHNgKABDAC4oM0trWykWNg5Mb1UNQMkwiKimeMSrGwlJNPZOLIALCBUVdHxcoA)?

I like this 👍🏼

Reply:
@jamesricky please change it to the proposed solution.

Reply:
Done in 2904df20fb3b6d5bf541830ca18298d5c4c5f0f7